.github/workflows/wasm.yml

```diff
@@ -2,28 +2,45 @@ name: Wasm
 
 on: [push, pull_request]
 
-# This will cancel previous runs when a branch or PR is updated
 concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.ref || github.run_id }}
   cancel-in-progress: true
 
 jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    outputs:
+      tag-name: ${{ steps.release.outputs.tag_name }}
+    steps:
+      - name: Run release-please
+        id: release
+        uses: google-github-actions/release-please-action@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          release-type: node
```

Comment:
We don't want to add release-please to this workflow. Instead, we want to update the current `release.yml` workflow so it will update the package.json version when it updates the cargo version.